test/client-test.js

```diff
@@ -500,7 +500,7 @@ var fs = require('fs'),
             assert.ok(result);
             assert.ok(client.lastResponse);
             assert.ok(client.lastResponseHeaders);
-            assert.ok(client.lastElapsedTime);
+            assert.ok(client.lastElapsedTime !== undefined);
```

Comment:
This seems to be flaky on my machine because it's 0 sometimes (and more likely with this change), which is falsy. However, we're only interested in the existence of the value.